bcauth/jinja2/account/email_confirmed.html

```diff
@@ -4,7 +4,6 @@
 {% block body_title %}{{ _("Confirm E-mail Address") }}{% endblock %}
 
 {% block content %}
-{% user_display confirmation.email_address.user as user_display %}
```

Comment:
I don't understand this line at all. The syntax suggests it's a Jinja2 tag, but no such tag existed. Then `user_display` gets used as a variable name and a global function. I figured this line should get removed.

Does that sound right?